pages/Configuring/Multi-GPU.md

```diff
@@ -0,0 +1,132 @@
+# Table of contents
+
+{{< toc >}}
+
+# General
+
+If your host machine uses multiple GPUs, you may want to primarily use a GPU for rendering all the elements for Hyprland including windows, animations, hardware acceleration for certain applications, etc.
```

Comment:
I think in the case of mobile systems (laptops) it's wise to say that Hyprland should run on the integrated GPU, and any heavier tasks should be offloaded to the discrete GPU.